impeller/entity/geometry/circle_geometry.cc

```diff
@@ -66,10 +66,74 @@ GeometryResult PointFieldGeometry::GetPositionUVBuffer(
   };
 }
 
+VertexBuffer CircleGeometry::GetPositionBufferCPU(
+    const ContentContext& renderer,
+    const Entity& entity,
+    RenderPass& pass) {
+  if (radius_ < 0.0) {
+    return {};
+  }
+  auto determinant = entity.GetTransformation().GetDeterminant();
+  if (determinant == 0) {
+    return {};
+  }
+
+  Scalar min_size = 1.0f / sqrt(std::abs(determinant));
```

Comment:
min_size is appropriate for strokes (i.e. when this is created from DrawPoints), but not when we are filling an actual circle (i.e. when created from DrawCircle).